doc/releases/changelog-0.28.0.md

```diff
@@ -126,128 +206,43 @@
   ...
```

Comment:
Maybe `qml.FermionicSWAP` can go after the `CPhaseShift00 ...` entry, so that we have the two controlled ones together (super minor though)